pkg/manifest/processor.go

```diff
@@ -0,0 +1,45 @@
+package manifest
+
+import (
+	"log/slog"
+	"os"
+
+	"github.com/kartverket/skipctl/pkg/logging"
+)
+
+type Processor struct {
+	log      *slog.Logger
+	logLabel string
+}
+
+type StringToErrorFunc func(string) error
+
+func NewManfiestProcessor(label string) *Processor {
+	return &Processor{
+		log:      logging.Logger(),
+		logLabel: label,
+	}
+}
+
+func (p *Processor) ProcessManifests(files []string, process StringToErrorFunc) {
+	failed := false
+
+	for _, file := range files {
+		err := process(file)
+
+		if err != nil {
+			p.log.Error(p.logLabel+" failed",
```

Comment:
```suggestion
			p.log.Error(p.logLabel + " failed",
```
Litt mer spacing er fint